conf.d/spark.yaml.example

```diff
@@ -0,0 +1,24 @@
+init_config:
+
+instances:
+  #
+  # The Spark check retrieves metrics from YARN's ResourceManager. This
+  # check must be run from the Master Node and the ResourceManager URI must
+  # be specified below. The ResourceManager URI is composed of the 
+  # ResourceManager's hostname and port.
+  #
+  # The ResourceManager hostname can be found in the yarn-site.xml conf file
+  # under the property yarn.resourcemanager.address
+  #
+  # The ResourceManager port can be found in the yarn-site.xml conf file under
+  # the property yarn.resourcemanager.webapp.address
+  #
+  - resourcemanager_uri: http://localhost:8088
+
+  # An optional friendly name can be specified for the cluster.
+  #   cluster_name: MySparkCluster
```

Comment:
nitpick: could you add two spaces of indentation on these two lines (given that they're at the same level as the `tags`, i.e. at the instance level)